src/main/scripts/travis-build.bash

```diff
@@ -44,7 +44,7 @@ function main() {
             return 1
         fi
         if ! wget https://api-staging.atomist.services/model/schema -O src/main/resources/com/atomist/rug/ts/cortex.json; then
-            err "Failed to download production cortex json"
+            err "Failed to download staging cortex json"
```

Comment:
Should we always download staging? Perhaps on release we should be downloading the production version?